portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -34,8 +34,11 @@
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
-  private DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
+  private final DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
+<<<<<<< HEAD
```

Comment:
What's this?

Reply:
probably from one of the merge conflicts.. fixed this